build-tools/debian-metadata/oss-xamarin.android-targets.install

```diff
@@ -0,0 +1,11 @@
+bin/Release/bin/*					/usr/bin
```

Comment:
> Do you want me to try and make those changes, or are they unrelated from this PR?

A bit of both?

The questions are twofold:

1. Which tools should be placed into `/usr/bin`?
2. What build system/organizational changes are required to support (1)?

This PR is a good "sounding board" to discuss those questions.

I believe that the only tool that should be in `/usr/bin` is `xabuild`.

I believe that the other tools need to be "available", but (1) don't need to be in `$PATH`, and (2) *shouldn't* be in `$PATH`.

That said, I don't fully understand the FlatPak and Debian directory structures, so I'm not sure where "non-public programs" should go, and what build system changes may be required to support that.